regression/cbmc-java/phi-merge_uninitialized_values/local.desc

```diff
@@ -0,0 +1,9 @@
+CORE 
+PhiMergeUninitialized.class
+--function PhiMergeUninitialized.dynamicAllocationUninitialized --show-vcc
+activate-multi-line-match
+^EXIT=0$
+^SIGNAL=0$
+--
+^.*: (dynamic_object|new_tmp[0-9]+(@[0-9]+)?)[0-9]+#0\)
+^.*\? (dynamic_object|new_tmp[0-9]+(@[0-9]+)?)[0-9]+#0
```

Comment:
In test.desc files a second `--` delimits comments. You should add one and then comment on what these regexes are looking for.

Format example:

```
CORE
Main.class
--other --arguments
Must find this
Also must find this
--
Musn't find this
Nor this
--
This is a comment
This is another comment
```